src/components/search/PageSearchModal/index.js

```diff
@@ -119,7 +119,7 @@ module.exports = React.createClass({
     if(this.props.onSelect) {
       this.props.onSelect(page);
     } else {
-      window.location = page.url;
+      return null;
```

Comment:
You could also just delete this whole `else`, returning null suggests to me that the return value here is useful. I'm not sure that the `null` is being used by anything is it?